pywt/_multidim.py

```diff
@@ -276,7 +277,7 @@ def idwtn(coeffs, wavelet, mode='symmetric', axes=None):
     return coeffs['']
 
 
-def swt2(data, wavelet, level, start_level=0):
+def swt2(data, wavelet, level, start_level=0, axes=(-2, -1)):
```

Comment:
This mirrors ``dwtn``, but it's actually a bit superfluous right? If you have an n-D array on which you want to do a transform over 2 axes, what's the advantage of using ``swt2`` over ``swtn``?